src/certificates/certificates-card/CertificatesCard.test.jsx

```diff
@@ -0,0 +1,104 @@
+import { render, waitFor, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+
+import initializeStore from '../../store';
+import { getComponentMode } from '../data/selectors';
+import { MODE_STATES } from '../data/constants';
+import { createCourseCertificate } from '../data/thunks';
+import CertificatesCard from './CertificatesCard';
+import messages from './messages';
+
+let store;
+const courseId = 'course-123';
+const certificateId = 1;
+const signatories = [{
+  name: 'John Doe', title: 'CEO', organization: 'Company', signatureImagePath: '',
+}];
```

Comment:
[question]: We already have a mock of this data in `src/certificates/__mocks__/signatories.js`, can we reuse it here?